### ISSUE TITLE
Accessibility for more info on challenged proposal (review)

### ISSUE THREAD

**Issue by divine-comedian** [1Hive/gardens#268]
My proposal was challenged but I'm not sure why - the only way to see the reason for being challenged (from the challenger) is by clicking `Accept Settlement`.

As a new user I'm probably not going to want to Accept Settlement or do any action until I have all the information on why I'm being challenged. Also the header for this explanation is `Argument in favour of cancelling action` which is a bit odd - Suggesting `Reason for Challenge` in place of that.
A few simple suggestions would be a button or bolded text saying `Why am I being Challenged?` which would pop-up the `Reason for Challenge` text. 
Another solution could be a hover modal on the `Challenged` status on the proposal card or you could add the `Reason for Challenge` somewhere directly on the page  

for context this is my current view as a user being challenged - 
![image](https://user-images.githubusercontent.com/67759413/129235266-0d26004c-c465-4913-90ba-ce80696de607.png)

**Comment by 0xGabi** [comment]
We should build from the new designs: https://www.figma.com/file/IGMjM4UyW2yp9ImEUsHduG/✨Improvements-based-on-UX-research?node-id=0%3A1

**Comment by rperez89** [comment]
@0xGabi seeing on the desings that there is some kind of conversation added to the arguments part introduced on the new designs , also some like heart, is this intended to be done with ceramic or what is the idea? 
<https://www.figma.com/file/IGMjM4UyW2yp9ImEUsHduG/%E2%9C%A8Improvements-based-on-UX-research?node-id=5%3A1187>

**Comment by 0xGabi** [comment]
Yes, most likely we will be using Ceramic + IDX to store the off-chain data related to a proposal and the challenge arguments.

**Comment by divine-comedian** [comment]
testing this on the rinkeby deployment - 

I can see there has been the show argument panel added under the bottom of the main proposal panel. I see however there is still no functionality to reply to a challenge argument according to the figma spec. 

Also I would suggest that default state for the argument panel should be open, currently when you land on the page it is hidden - we should make the arguments accessible and obvious for users. This is the current view on page load  - 

![image](https://user-images.githubusercontent.com/67759413/134541424-c81d1bd1-6a4d-49da-ba06-b429fd0ee180.png)
